src/probability/independence.lean

```diff
@@ -225,6 +235,15 @@ lemma indep_sets.Inter [measurable_space Ω] {s : ι → set (set Ω)} {s' : set
   indep_sets (⋂ n, s n) s' μ :=
 by {intros t1 t2 ht1 ht2, cases h with n h, exact h t1 t2 (set.mem_Inter.mp ht1 n) ht2 }
 
+lemma indep_sets.bInter [measurable_space Ω] {s : ι → set (set Ω)} {s' : set (set Ω)}
+  {μ : measure Ω} {p : ι → Prop} (h : ∃ n (hn : p n), indep_sets (s n) s' μ) :
+  indep_sets (⋂ n (hn : p n), s n) s' μ :=
```

Comment:
Can you phrase the lemma as `⋂ n ∈ I, s n` for some `I : set ι` instead? I think most `bInter` lemmas are of this form